study/cyber-tutor-2/js/app.js

```diff
@@ -0,0 +1,204 @@
+    correctAnswer: "Patching",
+	responseVal: false
+  },
+  {
+    question: "A hacker realizes that Julie's system has a vulnerability; he takes advantage of it using a program. What is this program called?",
+    correctAnswer: "Exploit",
+	responseVal: false
+  },
+  {
+    question: "Henry goes on nvd.nist.gov. What does he find?",
+    correctAnswer: "CVEs",
+	responseVal: false
+  },
+];
+
+/*const ALL_PROBLEM_TYPES = [
+    "forLoop",
+    "nestedForLoop",
+    "whileLoop"
+];*/
+
+// globals
+let score = 0;
+//let selectedProblemTypes = JSON.parse(JSON.stringify(ALL_PROBLEM_TYPES));
+let problem, answer;
+let number = 0;
+// initialize UI components
+
+showProblem();
+
+setAnswer();
+//createProblemTypeCheckboxes();
+wordBank();
+// bind onclick functions to the buttons
+document.getElementById("submit").onclick = checkAnswer;
+
+function showProblem(){
+	problem = myQuestions[number].question;
+ // show problem text to user
+    updatePrompt(problem);
+
+}
+function setAnswer(){
+	answer = myQuestions[number].correctAnswer;
+}
+
+
+/*function createProblemTypeCheckboxes() {
+    // create checkboxes for the user to be able to customize problem types that they practice
+    ALL_PROBLEM_TYPES.forEach(problemType => {
+        // create list item element
+        const li = document.createElement("li");
+
+        // create a checkbox element
+        const input = document.createElement("input");
+        input.type = "checkbox";
+        input.id = problemType;
+        input.name = problemType;
+        input.value = problemType;
+        input.checked = true;
+
+        // define click action to add/remove problem type from selected problem types
+        input.onclick = () => {
+            // get the value of the input element
+            const val = input.value;
+
+            // if the input is checked, add to list of selected problem types
+            if (input.checked) {
+                selectedProblemTypes.push(val);
+            }
+            // if input is unchecked, remove it from the list of selected problem types
+            else {
+              //  selectedProblemTypes.splice(selectedProblemTypes.indexOf(val), 1);
+            }
+
+            // update prompt
+            promptText = showProblem();//generateProblem();
+            updatePrompt(promptText);
+        }
+
+        // create label for checkbox input
+        const label = document.createElement("label");
+       // label.htmlFor = problemType;
+       // label.innerText = problemType;
+
+        // add checkbox and label to list item
+        li.appendChild(input);
+        li.appendChild(label);
+
+        // add list item to the problem types list element
+        //document.getElementById("problemTypes").appendChild(li);
+    });
+}
+*/
+
+
+function updatePrompt(code) {
```

Comment:
In order to stop any weird syntax highlighting from happening, you should remove the CodeMirror function calls and it should be more like this:
```js
function updatePrompt(code) {
    const codeEl = document.getElementById("code");

    codeEl.innerHTML = code;
}